myscript.js

```diff
@@ -0,0 +1,51 @@
+// output function - BEGIN
+var outputArr = [];
+var outputLine = 0;
+var outputF = function (outputItem) {
+    outputArr [outputLine]=(outputItem);
```

Comment:
two comments here:
- you don't need space between array variable name and square brackets
- no need to add brackets around `outputItem`, so code above becomes:
`outputArr[outputLine] = outputItem;`